ui/modal_templates.go

```diff
@@ -81,45 +89,22 @@ func privacyInfo(th *decredmaterial.Theme) []layout.Widget {
 }
 
 func setupMixerInfo(th *decredmaterial.Theme) []layout.Widget {
+	text := `<div style="text-color: gray">
+				Two dedicated accounts (“mixed” & “unmixed”) will be created in order to use the mixer.
+			</div>
+			<h4>This action cannot be undone.</h4>`
```

Comment:
This text is actually bold and it should inherit the text size of the parent tag which means it should be inside the parent tag. Reference the design mockup for proper text styling.
```suggestion
			<b>This action cannot be undone.</b>`
```